t/t2410-sdexec-memlimit.t

```diff
@@ -51,7 +51,7 @@ EOT
 
 cat >getcg.sh <<EOT2
 #!/bin/sh
-CGDIR=/sys/fs/cgroup/\$(cat /proc/\$\$/cgroup | cut -d: -f3)
+CGDIR=/sys/fs/cgroup/\$(cat /proc/\$\$/cgroup | grep ^0: | cut -d: -f3)
```

Comment:
This still assumes the mount point of cgroups2  is `/sys/fs/cgroup`. On my system it is `/sys/fs/cgroup/unified`

```sh
$ grep cgroup2 /proc/mounts
cgroup2 /sys/fs/cgroup/unified cgroup2 rw,nosuid,nodev,noexec,relatime,nsdelegate 0 0
```

Reply:
This results in the error:

```
DelegateControllers=memory pids
cat: /sys/fs/cgroup//user.slice/user-1001.slice/session-4039.scope/cgroup.controllers: No such file or directory
1..0 # SKIP cgroups memory controller is not enabled
```

Note however, that the real cgroup.controllers appears to be empty on this system for my current cgroup, so the test would be skipped anyway.

Reply:
Ah yeah, your code that I borrowed from flux-security handles this case.  Hmm, I wonder if it would be ok to add a little built-in subcommand that can print this path, e.g. `flux cg path` or similar?  I do find myself building those paths by hand when manually poking at cgroups on installed flux.